lib/fetch/util.js

```diff
@@ -280,6 +280,23 @@ function tryUpgradeRequestToAPotentiallyTrustworthyURL (request) {
   // TODO
 }
 
+function _toUSVString (val) {
+  // TODO: This is internal to node core...
+  return val
```

Comment:
@jasnell can we expose this from node core or is there a JS/WASM implementation with the corresponding functionality?

Reply:
https://github.com/nodejs/undici/issues/981#issuecomment-901439727

Reply:
That won't work anymore unfortunately.